packages/@aws-cdk/aws-cloud9-alpha/README.md

```diff
@@ -135,9 +144,35 @@ new cloud9.Ec2Environment(this, 'C9Env', {
 })
 ```
 
+To specify a specific IAM Federated User as the environment owner, use `Owner.federatedUser(accountId, userName)`. The user should have the `AWSCloud9Administrator` managed policy
```

Comment:
```suggestion
To specify a specific IAM Federated User as the environment owner, use `Owner.federatedUser(accountId, userName)`. 
The user should have the `AWSCloud9User` (preferred) or `AWSCloud9Administrator` managed policy attached.
```
From [docs](https://docs.aws.amazon.com/cloud9/latest/user-guide/share-environment.html#share-environment-member-roles). Should be updated also in the rest of the README.

Reply:
Thanks. Resolved in [91d8984](https://github.com/aws/aws-cdk/pull/27001/commits/91d898486feaf9d87bd86bde8b2fb69fe9eae601)